docs/using-the-jdbc-driver/DataSource.md

```diff
@@ -64,6 +64,9 @@ To use the AWS JDBC Driver with a connection pool, you must:
    ds.addDataSourceProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
    ds.addDataSourceProperty("serverPort", "5432");
    ds.addDataSourceProperty("database", "postgres");
+   
+   // The AwsWrapperDataSource can also be configured with a JDBC URL.
+   ds.addDataSourceProperty("jdbcUrl", "jdbc:aws-wrapper:postgresql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/postgres");
```

Comment:
Did we test this? IIRC you can't set jdbcUrl and servername or ds class name at the same time. Can we make sure we call that out.

Reply:
I see that we have some explanation in the section below, can we either link to the section below for more info or copy pasta to have the same warning in the comments?

Also does it make sense to set the protocol, servername and the jdbcUrl in the same properties? It seems really redundant.
```
   ds.addDataSourceProperty("jdbcProtocol", "jdbc:postgresql:");
   ds.addDataSourceProperty("serverName", "db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com");
  ds.addDataSourceProperty("jdbcUrl", "jdbc:aws-wrapper:postgresql://db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com:5432/postgres");
```

Reply:
You can set both the jdbcUrl and servername in the datasource, but the url takes precedence. The line to set the jdbcUrl was added to show it was possible, not necessarily that they should all be used together.